components/reddit/reddit.app.js

```diff
@@ -0,0 +1,189 @@
+const axios = require("axios");
+const get = require("lodash.get");
+module.exports = {
+  type: "app",
+  app: "reddit",
+  propDefinitions: {
+    subreddit: {
+      type: "string",
+      label: "Subreddit",
+      description: "The subreddit you'd like to watch.",
+      optional: false,
```

Comment:
`optional: false` is the default, so you can remove it

Reply:
ok, noted.